config/config.go

```diff
@@ -300,7 +300,6 @@ type PersesConfig struct {
 	InternalURL    string            `yaml:"internal_url" json:"internalURL"` // replaces the old in_cluster_url
 	IsCore         bool              `yaml:"is_core,omitempty" json:"isCore,omitempty"`
 	Project        string            `yaml:"project,omitempty" json:"project,omitempty"`
-	URLFormat      string            `yaml:"url_format,omitempty" json:"urlFormat,omitempty"`
```

Comment:
I agree with this, I don't see it used in the code. Probably some sort of cut+paste issue when defining this new Perses config.